child_compassion/models/field_office.py

```diff
@@ -113,3 +117,62 @@ class FieldOfficeHighRisks(models.Model):
     res_field = 'high_risk_ids'
 
     value = fields.Char(translate=True)
+
+
+class FieldOfficeDisasterAlert(models.Model):
+    _name = 'fo.disaster.alert'
+    _description = 'FO disaster alert'
+    res_model = 'compassion.field.office'
+    res_field = 'disaster_alert_ids'
+
+    disaster_id = fields.Char(required=True)
+    area_description = fields.Char()
+    close_date = fields.Date()
+
+    disaster_name = fields.Char()
+    disaster_date = fields.Datetime()
+    disaster_status = fields.Char()
+    disaster_type = fields.Char()
+
+    estimated_basic_supplies_needed = fields.Char()
+    estimated_homes_destroyed = fields.Char()
+    estimated_loss_of_life = fields.Char()
+    estimated_loss_of_livelihood = fields.Char()
+    estimated_not_attending_project = fields.Char()
+    estimated_rehabilitation_funds_usd = fields.Integer()
+    estimated_relief_funds_usd = fields.Integer()
+    estimated_serious_injuries = fields.Char()
+
+    field_office_name = fields.Many2many(
+        'compassion.field.office', 'country', 'Field Office'
+    )
+    field_office_damage = fields.Char()
+    field_office_impact_description = fields.Char()
+
+    impact_description = fields.Char()
+    impact_on_icp_infrastructure_damaged = fields.Integer()
+    impact_on_icp_infrastructure_destroyed = fields.Integer()
+    impact_on_icp_program_temporarily_closed = fields.Integer()
+    impact_to_field_office_operations = fields.Char()
+
+    is_additional_funds_requested = fields.Boolean()
+    is_communication_sensitive = fields.Boolean()
+    is_estimated_damage_over_one_million_usd = fields.Boolean()
+
+    reported_loss_of_life_beneficiaries = fields.Integer()
+    reported_loss_of_life_caregivers = fields.Integer()
+    reported_loss_of_life_siblings = fields.Integer()
+    reported_number_beneficiaries_impacted = fields.Integer()
+    reported_number_of_icps_impacted = fields.Integer()
+    reported_serious_injuries_beneficaries = fields.Integer()
+    reported_serious_injuries_caregivers = fields.Integer()
+    reported_serious_injuries_siblings = fields.Integer()
+    response_description = fields.Char()
+
+    severity_level = fields.Char()
+    source_kit_name = fields.Char()
+
+    # TODO: Implement and link these objects to the model
+    # beneficiary_disaster_impacts
+    # disaster_communication_updates
+    # icp_disaster_impacts

```

Comment:
I wasn't sure about the correctness of how I interpreted the task so I didn't implement these objects just yet.